docs/SPEC.md

```diff
@@ -325,6 +325,15 @@ Interpolation example:
 
 [IMPL#1775](https://github.com/influxdata/flux/issues/1775) Interpolate arbitrary expressions in string literals
```

Comment:
This line can be removed with this change as it now implements arbitrary interpolation.